projects/components/src/sequence/sequence-chart.service.ts

```diff
@@ -101,6 +101,9 @@ export class SequenceChartService {
       },
       onSegmentHovered: () => {
         /** NOOP */
+      },
+      onMarkerHovered: () => {
```

Comment:
why not `angular.noop` ?

Reply:
is there angular.noop in latest version? Not sure, 
It was already done in say way for previous funcs as well, So copied